pkg/util/oc/pod_commands.go

```diff
@@ -83,3 +90,13 @@ func WaitAllPodsReady(t test.TestHelper, ns string) {
 	t.T().Helper()
 	shell.Executef(t, `oc -n %s wait --for condition=Ready --all pods --timeout 180s`, ns)
 }
+func WaitCondition(t test.TestHelper, ns string, kind string, name string, condition string) {
```

Comment:
Missing vertical space between functions. 

We need to add a linter for these things.